plugins/invisible-typing/src/index.ts

```diff
@@ -12,7 +12,7 @@ export interface SettingsType {
   button?: boolean;
   invisible?: boolean;
   channelWise?: boolean;
-  channels?: Record<string, boolean>;
+  channels?: Record<string, boolean | undefined>;
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Allowing 'undefined' in 'channels' type definition aligns with the nullish coalescing usage.

```suggestion
  channels?: Record<string, boolean | undefined>;
```